src/site/markdown/threading.md

```diff
@@ -88,6 +93,27 @@ call, as there is an internal mutex that is locked and unlocked in order to ensu
 only one thread can be started at a time.  Failure to do this may lead to deadlock.
 The ThreadUtility::configure method handles this automatically.
 
+### Configuring Thread {#configuring}
+
+To tell Log4cxx what to do by default when starting a new thread, the enum
+[ThreadConfigurationType](@ref log4cxx.helpers.ThreadConfigurationType) may be
+used to configure the library appropriately.  By default, all signals on POSIX
+systems will be blocked to ensure that other threads do not get signals.
+
+To change this default, a simple change to your configuration files may be done.
+
+Example with XML configuration:
+```
+<log4j:configuration threadConfiguration="BlockSignalsOnly">
```

Comment:
Opinion: As `BlockSignalsOnly` is the default already and you talk about changing that, how about using `NameThreadOnly` or some other option as example instead? That's what I expected after reading your text. As you have two examples, you could as well use two different keywords.

Reply:
Changing it to be the not default makes sense to me.  The reasoning behind having two examples with the same setting is that the only difference is the config file format, you're not changing two things at a time.